### ISSUE TITLE
Support bookshelf ORM

### ISSUE THREAD

**Issue by jamesjwarren** [jaumard/trailpack-passport#138]
Would you accept a PR to add support for [trailpack-bookshelf](https://github.com/trailsjs/trailpack-bookshelf)?

**Comment by jaumard** [comment]
If it's doesn't include too much specific stuff to maintain no problem to support an other ORM, normally only models should be added and it should work out of the box, if not this means the trailpack-bookshelf need some modifications.

**Comment by jamesjwarren** [comment]
Yes I thought I could just add the models to `api/models/bookshelf/` but it seems trailpack-bookshelf isn't compatible with the model interface used in `api/models/User` and `api/models/Passport`. The arguments passed into the config and schema methods don't appear to be standard across different trails model implementations: 
```js
static config(app, orm) {}
static schema(app, orm) {}
```
